tests/v2/test_1508-awkward-from-rdataframe.py

```diff
@@ -0,0 +1,83 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import pytest  # noqa: F401
+import numpy as np  # noqa: F401
+import awkward as ak  # noqa: F401
+
+import awkward._v2._lookup  # noqa: E402
+import awkward._v2._connect.cling  # noqa: E402
+
+ROOT = pytest.importorskip("ROOT")
+
+
+compiler = ROOT.gInterpreter.Declare
+
+
+def test_data_frame_vec_of_vec_of_integers():
+    ak_array_in = ak._v2.Array([[[1], [2]], [[3], [4, 5]]])
+
+    data_frame = ak._v2.to_rdataframe({"x": ak_array_in})
+
+    assert data_frame.GetColumnType("x").startswith("awkward::ListArray_")
+
+    ak_array_out = ak._v2.from_rdataframe(
+        data_frame,
+        column="x",
+    )
+    assert ak_array_in.to_list() == ak_array_out["x"].to_list()
+
+
+def test_data_frame_NumpyArray():
+    array = ak._v2.contents.numpyarray.NumpyArray(
+        np.array([0.0, 1.1, 2.2, 3.3]),
+        parameters={"some": "stuff", "other": [1, 2, "three"]},
+    )
+
+    layout = array
+    generator = ak._v2._connect.cling.togenerator(layout.form, flatlist_as_rvec=False)
+    lookup = ak._v2._lookup.Lookup(layout)
+    generator.generate(compiler)
+
+    array_out = generator.tolayout(lookup, 0, ())
+
+    # data_frame = ak._v2.to_rdataframe({"x": array})
+    #
+    # assert data_frame.GetColumnType("x") == "double"
+    #
+    # ak_array_out = ak._v2.from_rdataframe(
+    #     data_frame, column="x", column_as_record=False
+    # )
```

Comment:
Is this test still valid? (If not, we can delete it, rather than comment it out.)

Reply:
done